docs/index.html

```diff
@@ -396,6 +396,16 @@ <h2 id="web-extensions-%2F-manifest.json" tabindex="-1">Web Extensions / manifes
 <td>A manifest permission is not supported for the extension manifest_version</td>
 </tr>
 <tr>
+<td><code>MANIFEST_PERMISSIONS_PRIVILEGED</code></td>
```

Comment:
```suggestion
<td><code>MANIFEST_PERMISSION_PRIVILEGED</code></td>
```

for consistency with the other names

Reply:
I'm ok with changing this name for consistency, maybe it may be even more consistent renamed as `MANIFEST_PRIVILEGED_PERMISSION` but I wanted to mention why I named is as `*_PERMISSIONS_*`:

- most of the other permissions rule named as `*_PERMISSION` are reporting issues related to a single permission
- whereas this one has to report one error for all privileged permissions being found, because the custom keyword is set on the property and gets the entire set of permissions and reports one linting issue for all privileged permissions found in that array

I'm not against renaming it form a plural into a singular form, but I wanted to point that out in case that may affect our decision about it.